Cargo.toml

```diff
@@ -16,7 +16,7 @@ num-integer = { version = "0.1.45", default-features = false, features = [
     "i128",
 ] }
 soroban-sdk = "20.4.0"
-soroban-token-sdk = "20.4.0"
+soroban-token-sdk = "20.3.1"
```

Comment:
Why the downgrade? Not to mention, last week there was another upgrade to 20.5.0.